plugins/woocommerce/src/Admin/Features/LaunchYourStore.php

```diff
@@ -300,6 +301,35 @@ private function is_mailpoet_connected() {
 		return 'valid' === $api_state['state'] && 200 === $api_state['code'];
 	}
 
+	/**
+	 * Track the template change.
+	 *
+	 * @param int    $post_id The post ID.
+	 * @param object $post The post object.
+	 * @param bool   $update Whether the post is being updated.
+	 */
+	public function maybe_track_template_change( $post_id, $post, $update ) {
+		// Check multiple fields to avoid false matches with non-WooCommerce templates.
+		if ( isset( $post->post_name ) &&
```

Comment:
We only check isset( $post->post_name ). Do we need to check other fields? Or maybe check if it's a WP_Post object?

Reply:
Great suggestion! I've updated the checks in 433d01d